src/conversions/chrono.rs

```diff
@@ -463,7 +658,7 @@ mod tests {
             );
             assert_eq!(
                 none.extract::<Utc>().unwrap_err().to_string(),
-                "TypeError: 'NoneType' object cannot be converted to 'PyTzInfo'"
+                "ValueError: expected datetime.timezone.utc"
```

Comment:
Is this error message stable against using the limited and unlimited API? (We run our unit tests against both.)

Reply:
Yes! I simplified slightly the unlimited API implementation to not downcast to `tzinfo` before checking equality with `timezone.utc`